src/nativestore/PropertyLink.cpp

```diff
@@ -126,7 +126,7 @@ unsigned int PropertyLink::insert(std::string name, const char* value) {
         property_link_logger.warn("Property key/name already exist key = " + std::string(name));
         return this->blockAddress;
     } else if (this->nextPropAddress) {  // Traverse to the edge/end of the link list
-        property_link_logger.log("Next Called", "info");
+        property_link_logger.info("Next Called");
```

Comment:
Lets remove this log line. It does not have any meaning.

Reply:
Removed

Reply:
Ok